docs/deployment-process.md

```diff
@@ -0,0 +1,60 @@
+# Build Process
```

Comment:
This file is named deployment process yet seems to refer almost entirely to the build process

Reply:
Updated this to be clearer